.github/workflows/rosa-cluster-auto-delete-on-schedule.yml

```
@@ -0,0 +1,26 @@
+name: ROSA Scheduled Delete
+
+on:
+  schedule:
+    - cron: '0 19 * * *' # Runs every day at 7 PM UTC
+
+jobs:
+
+  delete:
+    name: Delete ROSA cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Add the following to prevent scheduled run in forks:

```suggestion
        uses: actions/checkout@v3
        if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
```

---

thank you, I will add this change